api/v1beta1/glance_types.go

```diff
@@ -186,3 +186,27 @@ func (instance Glance) IsReady() bool {
 	// - Both the internal and external API endpoints are ready (count >= 1)
 	return instance.Status.GlanceAPIInternalReadyCount >= 1 && instance.Status.GlanceAPIExternalReadyCount >= 1
 }
+
+// GlanceExtraVolMounts exposes additional parameters processed by the glance-operator
+// and defines the common VolMounts structure provided by the main storage module
+type GlanceExtraVolMounts struct {
```

Comment:
@stuggi This part that was previously in `lib-common` is now versioned as part of the glance `v1beta1` API. Is this good enough?